rpms/python-kombu/python-kombu.spec

```diff
@@ -1,45 +1,25 @@
-%global srcname kombu
-
-
-Name:           python-%{srcname}
-# The Fedora package is using epoch 1, so we need to also do that to make sure ours gets installed
-Epoch:          1
-Version:        3.0.33
-Release:        7.pulp%{?dist}
-BuildArch:      noarch
+%{?scl:%scl_package python-kombu}
+%{!?scl:%global pkg_name %{name}}
 
-# utils/functional.py contains a header that says Python
-License:        BSD and Python
-Summary:        AMQP Messaging Framework for Python
-Group:          Development/Languages
-URL:            http://pypi.python.org/pypi/%{srcname}
-Source0:        http://pypi.python.org/packages/source/k/%{srcname}/%{srcname}-%{version}.tar.gz
-Patch0:         563.patch
-Patch1:         569.patch
-Patch2:         571.patch
-Patch3:         577.patch
-Patch4:         2124.patch
+%global srcname kombu
 
-BuildRequires:  python-anyjson
-BuildRequires:  python-nose
-BuildRequires:  python-setuptools
-BuildRequires:  python2-devel
+Name:		%{?scl_prefix}python-kombu
+Version:	4.0.2
+Release:	2%{?dist}
+Summary:	An AMQP Messaging Framework for Python
 
-# required for tests:
-BuildRequires: python-amqp >= 1.4.9
-BuildRequires: python-mock
-BuildRequires: python-qpid
-BuildRequires: python-qpid-qmf
-BuildRequires: python-simplejson
-BuildRequires: python-unittest2 >= 0.8.0
-BuildRequires: python2-msgpack
-BuildRequires: PyYAML
-BuildRequires: qpid-tools
+License:	BSD and Python
+URL:		http://pypi.python.org/pypi/%{srcname}
+Source0:	http://pypi.python.org/packages/source/k/%{srcname}/%{srcname}-%{version}.tar.gz
 
-Requires: python-amqp >= 1.4.9
-Requires: python-amqp < 2.0
-Requires: python-anyjson >= 0.3.3
+BuildRequires:  %{?scl_prefix_python}python-devel
+BuildRequires:  %{?scl_prefix_python}python-setuptools
+BuildRequires:  %{?scl_prefix}python-anyjson
+%{?scl:BuildRequires:	%{scl}-build %{scl}-runtime}
 
+Requires:  %{?scl_prefix}python-anyjson >= 0.3.3
```

Comment:
it's a relic from kombu3: https://github.com/celery/kombu/blob/v3.0.37/requirements/default.txt